### DISCUSSION TITLE
Problem run powershell script or cmd command

### DISCUSSION THREAD

**Discussion by Raito00** [Ylianst/MeshCentral#6310] [Issues]
Hello!

I have silmpe PowerShell command to set long date format:
```shell
 
   $dateFormatPath = "HKCU:\Control Panel\International"
   $newLongDateFormat = "dddd, MMMM d, yyyy"
   Set-ItemProperty -Path $dateFormatPath -Name "sLongDate" -Value $newLongDateFormat
   RUNDLL32.EXE USER32.DLL,UpdatePerUserSystemParameters
   
``` 
Or can use normal CMD
```
powershell -Command "$dateFormatPath = 'HKCU:\Control Panel\International'; $newLongDateFormat = 'dddd, MMMM d, yyyy'; Set-ItemProperty -Path $dateFormatPath -Name 'sLongDate' -Value $newLongDateFormat; RUNDLL32.EXE USER32.DLL,UpdatePerUserSystemParameters"

``` 
When i use Terminal -> User Shell or User PoweShell
commands working
![image](https://github.com/user-attachments/assets/66d94867-95bc-42c8-9f5b-c343b8dbb1c1)

Problem: I need run command to all clients at once. If i use Run commands option - NOT WOORKING  :(
![image](https://github.com/user-attachments/assets/1c9fc54c-6ad1-4fba-a16a-6a7e8a7279ed)

**Comment by si458** [comment]
once you have run from the run commands,
check the output in the console tab and see if it shows any errors
chances are its the formatting thats incorrect

**Comment by Raito00** [comment]
From console if use Run Commands:
```
$dateFormatPath = "HKCU:\Control Panel\International"
$newLongDateFormat = "dddd, MMMM d, yyyy"
Set-ItemProperty -Path $dateFormatPath -Name "sLongDate" -Value $newLongDateFormat
RUNDLL32.EXE USER32.DLL,UpdatePerUserSystemParameters
Run commands completed.
``` 
But nothing happening ... 

if use Terminal User PoweShell -- Command working OK

**Reply by si458** [reply]
sorry just remembered! yes the is a problem with the `run commands` and `run as user`
basically it never runs under the users cmd.exe, it runs under the `system` cmd!
its a bug we are still trying to fix!
